packages/react/src/components/MenuButton/SplitMenuButton.jsx

```diff
@@ -37,11 +37,30 @@ const propTypes = {
    * be read by screen readers
    */
   iconDescription: PropTypes.string.isRequired,
+
+  /**
+   * The size of the button and the dropdown items
+   */
+  size: PropTypes.oneOf(['sm', 'md', 'default']).isRequired,
```

Comment:
Do we need to make these required? Can we add a default and make this optional? Same comment with `kind`.

Reply:
SplitMenuButton.jsx is an internal component and MenuButton is the only consumer, so I made the size prop required in order to not duplicate the default value. But I can add a default if you think that makes more sense?

Reply:
No, if its internal only there is no need.